doc/reference/configuration.rst

```diff
@@ -925,9 +817,7 @@ FilePathResolverExtension
 """""""""""""""""""""""""""""""""""
 
 
-
-
-**Default**: ``"\/home\/daniel\/www\/phpactor\/phpactor"``
+**Default**: ``"\/home\/mamazu\/packages\/phpactor\/phpactor"``
```

Comment:
Well, this is also something that we might want to parameterize in the documentation.

Reply:
fine to leave it i you are, otherwise feel free to change to `daniel` :smile:

Reply:
Well the thing is that every time you run composer integrate it generates it again so I always have to be careful not to commit it.

Reply:
like i say, i don't mind if it's committed tbh feel free to keep it as is or change it

Reply:
it will just get reverted to my path when i push the docs next :smile:

Reply:
Haha, yeah. All good. I am just irritated about things I didn't change. :D